services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalCertificatesStructureProvider.java

```diff
@@ -38,6 +45,7 @@
   public static final String DSCS = "dscs";
```

Comment:
I know it is not part of what you changed, but can we refactor also that name? What does it even mean?